libs/orchestrator-ui-components/src/lib/components/breadcrumb-navigation/breadcrumb-navigation.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { EuiBreadcrumb, EuiBreadcrumbs } from '@elastic/eui';
+import { NextRouter } from 'next/router';
```

Comment:
Hmm, we can't use Next Router from inside the lib.
Some users will want to use the lib without Next.

I will probably refactor this, after merge. So just a reminder.